Objects/structseq.c

```diff
@@ -565,14 +567,16 @@ PyStructSequence_NewType(PyStructSequence_Desc *desc)
     spec.name = desc->name;
     spec.basicsize = sizeof(PyStructSequence) - sizeof(PyObject *);
     spec.itemsize = sizeof(PyObject *);
-    spec.flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_HAVE_GC;
+    spec.flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_HAVE_GC | tp_flags;
     spec.slots = slots;
 
-    type = (PyTypeObject *)PyType_FromSpecWithBases(&spec, (PyObject *)&PyTuple_Type);
+    type = (PyTypeObject *)PyType_FromSpecWithBases(&spec,
+        (PyObject *)&PyTuple_Type);
```

Comment:
Why do you use ``PyType_FromSpecWithBases`` instead of ``PyType_FromModuleAndSpec`` here?

```suggestion
    type = (PyTypeObject *)PyType_FromModuleAndSpec(
        module, &spec, (PyObject *)&PyTuple_Type);
```

Reply:
Hi, Christian. Thanks for your review.
I copy the codes from `PyStructSequence_NewType()` to `PyStructSequence_FromModuleAndDesc()` :)
The previous code have use `PyType_FromSpecWithBases` in https://github.com/python/cpython/blob/main/Objects/structseq.c#L571.
Compared to `PyStructSequence_NewType()`, `PyStructSequence_FromModuleAndDesc()` can receive the `tp_flags`.

Reply:
Thanks for the explanation. I alread got that. That was not my point. :)

What I mean is that you could use ``PyType_FromModuleAndSpec`` to simplify the code. You would no longer have to set ``ht_module`` manually. I have updated my initial comment with a code suggestion.

Reply:
Oh, got it. You are right. This is a good suggestion.
Petr have another suggestion in https://bugs.python.org/issue45113#msg403728.
If we reach an aggrement with Petr, I will update this PR, Thanks.

Reply:
Thanks, updated.